canvas_modules/common-canvas/locales/palette/locales/en.json

```diff
@@ -1,3 +1,5 @@
 {
-  "palette.flyout.search.placeholder": "Find palette nodes"
+  "palette.flyout.search.placeholder": "Find palette nodes",
+  "palette.flyout.label": "Flyout Palette",
+  "palette.modal.label": "Modal Palette"
```

Comment:
Can you change this to be "Dialog Palette" -- it sin't really a Modal and was badly named in the first place